tox.ini

```diff
@@ -54,8 +54,10 @@ basepython =
     3.4: python3.4
     3.5: python3.5
     3.6: python3.6
+    3.7: python3.7
```

Comment:
If this `tox.ini` were to use the [tox default factors](https://tox.readthedocs.io/en/latest/config.html#factors), we could avoid specifying the `basepython` all together. For example `py27`, `py36`, `py37`, ...

See: https://tox.readthedocs.io/en/latest/config.html#confval-basepython=NAME-OR-PATH

Reply:
Explicit is better than implicit (at least until we get the failing tests to pass).